bin/load-env.sh

```diff
@@ -11,10 +11,18 @@ then
   exit 1
 fi
 
-if [ ! -f "$FLUO_PROPS" ]
-then
+if [ -z "$APP_NAME" ]; then
+  echo "APP_NAME is not set!" 
+  exit 1
+fi
+
+if [ ! -d $FLUO_HOME/apps/$APP_NAME ]; then
+  $FLUO_HOME/bin/fluo new $APP_NAME
```

Comment:
I added it here because there is a check below to see if $FLUO_PROPS exists (which will fail).  I agree through it should be moved out.  I can move this to the configure script.  And then add a $FLUO_PROPS check to each script that requires it.